### DISCUSSION TITLE
Information Architecture, Content and Brands Updates

### DISCUSSION THREAD

**Discussion by hackfisher** [darwinia-network/darwinia#1495] [General]
Many community members have noticed some changes in the website and the brand.

Recently, we invited PaperMoon to [propose](https://github.com/darwinia-network/collaboration/pull/58/files) and report on our technical education, website, and documentation content. We found  the recommendations in [their report](https://github.com/darwinia-network/collaboration/blob/master/service-providers/papermoon/202405%20-%20Darwinia%20Docs%20Final%20Report.pdf) to be valuable and should be adopted, particularly in aspects of market and audience orientation. It can help Darwinia and Msgport better organize content and express it more clearly. Content and docs education team adopted their suggestions and made changes and upgrades to the information structure, content, and branches in conjunction with the governance system and the [DIP-7](https://github.com/darwinia-network/DIP-7) upgrade.

To make these changes and details clearer to the community, here is a summary:

- [Darwinia](https://darwinia.network/) mainly refers to the Darwinia Chain, which is an EVM-compatible parachain on Polkadot. It provides and integrates a cross-chain stack, including Msgport, to facilitate application developers in developing cross-chain applications. Currently, it is primarily positioned as a cross-chain relay station between Polkadot and EVM chains. (Before this report, our branding strategy was more inspired by Polygon, aiming to upgrade Darwinia’s brand to include cross-chain middleware and services like Msgport. However, this report pointed out that such a brand upgrade is very challenging. Many partners and users in the ecosystem have a solidified brand impression of Darwinia, making it difficult to change. This would lead to a lot of content and conceptual confusion, making it hard to understand. Therefore, the report suggests separating Msgport from the information architecture, content, and brand).

- Splitting [Msgport](https://msgport.xyz/) into an independent brand, while it will increase maintenance workload, is still worthwhile considering the clarity for users and the market.

- RingDAO refers to the next governance version of Darwinia, or it can refer to the DIP-7 currently being developed. This governance system can govern not only the Darwinia Chain but also capture new projects and value through token governance. The separated Msgport will primarily connect and coordinate with Darwinia through RingDAO. Since the project's value comes from users and the market, this arrangement can also facilitate a certain degree of decoupling. Msgport, as a cross-chain intermediary service layer, does not necessarily depend on the operation of the Darwinia Chain. A certain degree of detachment will reduce the complexity for new users to understand. Through RingDAO, they can achieve governance and value binding between them.

```
                                             +-------------+                                                
                                             |             |                                                
        +------------------------------------+   RingDAO   +---------------------------------------+        
        |                                    |             |                                       |        
        |                                    +------+------+                                       |        
        |                                           |                                              |        
        |                                           |                                              |        
        |                                           |                                              |        
        |                                           |                                              |        
        |                                           |                                              |        
        |                                           |                                              |        
        |                                           |                                              |        
        |                                           |                                              |        
        |                                           |                                              |        
+-------v---------+                        +--------v---------+                           +--------v-------+
|                 |                        |                  |                           |                |
|    Darwinia     |                        |       Msgport    |                           |      XAPI      |
|                 |                        |                  |                           |                |
+-----------------+                        +------------------+                           +----------------+
```

**Comment by hackfisher** [comment]
Website updated for darwinia.network: https://darwinia.network/

**Comment by johneggleston** [comment]
Solid work!
